x-pack/test/apm_api_integration/basic/tests/traces/top_traces.ts

```diff
@@ -53,12 +54,307 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           'impact'
         );
 
-        const expectedTracesWithoutSamples = sortBy(
-          expectTopTraces.map((item: any) => omit(item, 'sample')),
-          'impact'
-        );
-
-        expect(responseWithoutSamples).to.eql(expectedTracesWithoutSamples);
+        expectSnapshot(responseWithoutSamples).toMatchInline(`
```

Comment:
I'll probably remove this later, this is just so we can see what it would mean to use inline snapshots for bigger responses.

Reply:
Replaced it with a more realistic scenario.